src/ja/server/scheduler/scheduler.py

```diff
@@ -0,0 +1,30 @@
+from ja.server.database.database import ServerDatabase
+from ja.server.scheduler.algorithm import SchedulingAlgorithm
+
+
+class Scheduler:
+    """
+    Scheduler is a class which is responsible for generating the new job
+    distribution whenever the database changes.
+
+    Internally it uses a SchedulingAlgorithm to determine how to schedule
+    jobs which are "alive".
```

Comment:
"Alive" is not precise enough I think.

Reply:
is `runnable` better? This is what I have used below in `reschedule`